feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in feature1 branch
```

Comment:
It's really good that you have decided to add this code

Reply:
thanks